README.md

```diff
@@ -578,7 +591,12 @@ The second argument handles the rejection case of the promise and maps the error
 **Signature:**
 
 ```typescript
-fromPromise<U, E>(p: Promise<U>, f: (e: unknown) => E):  ResultAsync<U, E> { ... }
+class ResultAsync<T, E> {
+  fromPromise<U, E>(
+    promise: Promise<U>,
+    callback: (unknownError: unknown) => E)
+  ): ResultAsync<U, E> { ... }
+}
```

Comment:
Because `fromPromise` is a static class method, it means that there isn't a "previous `Result<T, E>`" - so to speak. 

The signature should probably look something like this to better reflect the fact that you're not working with a pre-existing `T` or `E` type:

```typescript
// fromPromise is a static class method
//
// also available as a standalone function
// import { fromPromise } from 'neverthrow'
ResultAsync.fromPromise<T, E>(
  promise: Promise<T>,
  errorHandler: (unknownError: unknown) => E)
): ResultAsync<U, E> { ... }
```

Also, using `callback` to refer to the error handler is a little misleading since that function is only called when the underlying `Promise` rejects.